### DISCUSSION TITLE
Ability to group channels in the channel listing

### DISCUSSION THREAD

**Discussion by ricklecoat** [ExpressionEngine/ExpressionEngine#2946] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
Site editors confronted with a long list of channels might struggle to identify the correct one. This is especially true when some channels might focus on page text for site sections, others might be devoted to file management, and others might contain utilities — “switches’, if you like, used to control aspects of the site.

**Describe the solution you'd like**
If there was an optional way to ‘clump’ channels into custom-defined groups, that might help, especially in contexts such as the 'Entries' fly-out menu in EE6. Perhaps they could be implemented something like category groups or field groups. I see the end result displaying in a way something akin to an `<optgroup>` inside a `<select>` element. 

**Describe alternatives you've considered**
I can't think an alternative other than leaving the list of channels as it is — one big pot.

- [ ] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by benoitlemay** [comment]
At least if we could reorder the channel listing!

**Comment by PaulBaileyAtChan** [comment]
I like this. I have a lot of channels knocking around, and tend to very clumsily group them with their names; a more standard way of grouping them, under custom group names, would be a nice addition.

**Comment by jaredchesebro** [comment]
I think the minimum viable product for this would be just the ability to reorder channels similar to how you can reorder categories within category groups. A nice drag and drop interface.
